labelme/app.py

```diff
@@ -658,6 +660,14 @@ def loadFile(self, filename=None):
             if LabelFile.isLabelFile(filename):
                 try:
                     self.labelFile = LabelFile(filename)
+                    # FIXME: PyQt4 installed via Anaconda fails to load JPEG
+                    # and JSON encoded images.
+                    # https://github.com/ContinuumIO/anaconda-issues/issues/131
+                    if QImage.fromData(self.labelFile.imageData).isNull():
+                        raise LabelFileError(
+                            'Failed loading image data from label file.\n'
+                            'Maybe this is a known issue of PyQt4 built on'
+                            ' Anaconda, and may be fixed by installing PyQt5.')
```

Comment:
@syan1992 I hope this PR fixes your problem in #17. But I failed to reopen the exported json file when using conda pyqt4 on my macOS. So if you use conda on windows, you may need to install pyqt5 instead.